tests/RobotFramework/tests/cumulocity/log/log_operation.robot

```diff
@@ -30,13 +30,26 @@ Request with non-existing log type
     ...    failure_reason=.*No such file or directory for log type: example1
     ...    timeout=120
 
+Manual log operation request
+    Execute Command    sudo -u tedge mkdir -p /var/tedge/file-transfer/${DEVICE_SN}/log_upload
+    Execute Command    sudo -u tedge touch /var/tedge/file-transfer/${DEVICE_SN}/log_upload/example-1234
+    ${start_timestamp}=    Get Current Date    UTC    -24 hours    result_format=%Y-%m-%dT%H:%M:%SZ
+    ${end_timestamp}=    Get Current Date    UTC    +60 seconds    result_format=%Y-%m-%dT%H:%M:%SZ
+    Execute Command
+    ...    sudo tedge mqtt pub --retain 'te/device/main///cmd/log_upload/example-1234' '{"status":"init","tedgeUrl":"http://127.0.0.1:8000/tedge/file-transfer/${DEVICE_SN}/log_upload/example-1234","type":"example","dateFrom":"${start_timestamp}","dateTo":"${end_timestamp}","searchText":"first","lines":10}'
+    ${messages}=    Should Have MQTT Messages
+    ...    te/device/main///cmd/log_upload/example-1234
+    ...    minimum=3
+    ...    maximum=3
```

Comment:
I'd rather add the assertion for the clear message with `minimum=0` and `maximum=0`. That'd be clearer I feel.

Reply:
Though it is a bit hard to check for the absence of a message when using the mqtt-logger (the historical record of all mqtt messages which are sent, which is used by they keyword). However I'll rewrite the test as follows:

1. Publish local cmd (with status="init")
2. Wait for topic to be updated with a status="successful"
3. Wait 5 seconds (a fixed period which is assumed that if the mapper has not reacted within 5 seconds then it will never react on the message)
4. Check that the last retained message on the topic still exists and status="successful"

Reply:
Resolved by 673135cac7ce86cecc4380699234141ead537675